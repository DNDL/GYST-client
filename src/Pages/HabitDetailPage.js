import React from 'react';
import Nav from '../components/Nav/Nav';
import Habit from '../components/Containers/Habit';

export default function HabitDetailPage(props) {
  return (
    <>
      <Habit props={props}/>
      <Nav />
    </>
  );
}
