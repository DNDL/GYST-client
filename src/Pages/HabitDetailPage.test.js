import React from 'react';
import { shallow } from 'enzyme';
import HabitDetailPage from './HabitDetailPage';

describe('HabitDetailPage component', () => {
  it('renders HabitDetailPage', () => {
    const wrapper = shallow(<HabitDetailPage props={{}}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
