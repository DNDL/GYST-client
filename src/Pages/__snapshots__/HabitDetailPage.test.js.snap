// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HabitDetailPage component renders HabitDetailPage 1`] = `
<Fragment>
  <Logo />
  <Connect(Habit)
    props={
      Object {
        "props": Object {},
      }
    }
  />
  <Nav />
</Fragment>
`;
